### PR TITLE
Swagger ui

### PR DESCRIPTION
**Pull request by Bromel777**

### CODE REVIEW

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/AddressInfo.scala [hunk 1]
```diff
@@ -76,9 +77,10 @@ object AddressInfo {
...
 
-  implicit private val txsValidator: Validator[Transactions] = Validator.derive
+  implicit private val txsValidator: Validator[Transactions] = Schema
+    .derived[Transactions].validator
 
-  implicit val schema: Schema[AddressInfo] = Schema.derive
+  implicit val schema: Schema[AddressInfo] = Schema.derived[AddressInfo]
 
-  implicit val validator: Validator[AddressInfo] = Validator.derive
+  implicit val validator: Validator[AddressInfo] = Schema.derived[AddressInfo].validator
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/AddressInfo.scala:85]: why deriving `Schema[AddressInfo]` twice?

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/OutputInfo.scala [hunk 2]
```diff
@@ -38,14 +37,14 @@ object OutputInfo {
       .modify(_.ergoTree)(_.description("Serialized ergo tree"))
       .modify(_.address)(_.description("An address derived from ergo tree"))
 
-  implicit val validator: Validator[OutputInfo] = Validator.derive
+  implicit val validator: Validator[OutputInfo] = Schema.derived[OutputInfo].validator
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/OutputInfo.scala:40]:
```suggestion
  implicit val validator: Validator[OutputInfo] = schema.validator
```

File: modules/explorer-core/src/main/scala/org/ergoplatform/explorer/db/models/EpochParameters.scala [hunk 3]
```diff
@@ -23,6 +23,6 @@ final case class EpochParameters(
 
 object EpochParameters {
 
-  implicit def schema: Schema[EpochParameters]       = Schema.derive[EpochParameters]
-  implicit def validator: Validator[EpochParameters] = Validator.derive[EpochParameters]
+  implicit def schema: Schema[EpochParameters]       = Schema.derived[EpochParameters]
+  implicit def validator: Validator[EpochParameters] = Schema.derived[EpochParameters].validator
```
Comment (oskin1) [inline modules/explorer-core/src/main/scala/org/ergoplatform/explorer/db/models/EpochParameters.scala:27]:
```suggestion
  implicit def validator: Validator[EpochParameters] = schema.validator
```

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/AddressInfo.scala (hunk 1 above)
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/AddressInfo.scala:85]:
```suggestion
  implicit val validator: Validator[AddressInfo] = schema.validator
```

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/ApiErr.scala [hunk 4]
```diff
@@ -35,13 +35,13 @@ object ApiErr {
...
+  implicit val unknownErrorS: Schema[UnknownErr] = Schema.derived[UnknownErr]
+  implicit val notFoundS: Schema[NotFound]       = Schema.derived[NotFound]
+  implicit val badInputS: Schema[BadRequest]     = Schema.derived[BadRequest]
 
-  implicit val unknownErrorV: Validator[UnknownErr] = Validator.derive[UnknownErr]
-  implicit val notFoundV: Validator[NotFound]       = Validator.derive[NotFound]
-  implicit val badInputV: Validator[BadRequest]     = Validator.derive[BadRequest]
+  implicit val unknownErrorV: Validator[UnknownErr] = Schema.derived[UnknownErr].validator
+  implicit val notFoundV: Validator[NotFound]       = Schema.derived[NotFound].validator
+  implicit val badInputV: Validator[BadRequest]     = Schema.derived[BadRequest].validator
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/ApiErr.scala:44]:
```suggestion
  implicit val unknownErrorV: Validator[UnknownErr] = unknownErrorS.validator
  implicit val notFoundV: Validator[NotFound]       = notFoundS.validator
  implicit val badInputV: Validator[BadRequest]     = badInputS.validator
```

File: build.sbt [hunk 5]
```diff
@@ -8,6 +8,8 @@ lazy val commonSettings = Seq(
   libraryDependencies ++= dependencies.Testing ++ dependencies.CompilerPlugins,
   test in assembly := {},
   assemblyMergeStrategy in assembly := {
+    case PathList("META-INF", "maven", "org.webjars", "swagger-ui", "pom.properties") =>
+      MergeStrategy.singleOrError
```
Comment (oskin1) [inline build.sbt:12]: How swagger ui got into jars at all? We don't use it, do we?

**Review by oskin1 — APPROVED** [review]

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/AddressInfo.scala (hunk 1 above)
Comment (Bromel777) [reply modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/models/AddressInfo.scala]: fix

File: build.sbt (hunk 5 above)
Comment (Bromel777) [reply build.sbt]: fix

**Pull request merged**